Duplicati/CommandLine/BackendTester/Duplicati.CommandLine.BackendTester.csproj

```diff
@@ -86,8 +86,12 @@
       <Project>{32a74526-3e5f-413a-8cb4-1efdad4c8b91}</Project>
       <Name>Duplicati.Library.Backend.Sia</Name>
     </ProjectReference>
-    <ProjectReference Include="..\..\Library\Backend\Tardigrade\Duplicati.Library.Backend.Tardigrade.csproj">
+    <ProjectReference Include="..\..\Library\Backend\Storj\Duplicati.Library.Backend.Storj.csproj">
       <Project>{ae035e01-c917-4f13-a35e-78f21c1a2f17}</Project>
+      <Name>Duplicati.Library.Backend.Storj</Name>
+    </ProjectReference>
+    <ProjectReference Include="..\..\Library\Backend\Tardigrade\Duplicati.Library.Backend.Tardigrade.csproj">
+      <Project>{ae035e01-c917-4f13-f35e-78f21c1a2f17}</Project>

```

Comment:
I'm surprised that the GUIDs for the Storj and Tardigrade projects differ by only one character.  Was this randomly generated?

Also, it appears that `Duplicati/Library/Backend/Tardigrade/Properties/AssemblyInfo.cs` contains Storj's GUID instead of Tardigrade's GUID?

Reply:
I copied the whole folder over instead of creating a new one and just changed one character, yes. I hope this is not a problem?
I did not see the GUID in the AssemblyInfo, though. I've changed it accordingly.

Reply:
> just changed one character, yes. I hope this is not a problem?

I'm not at all familiar with GUID usage here, but if this was the change to ae035e01-c917-4f13-f35e-78f21c1a2f17 from ae035e01-c917-4f13-a35e-78f21c1a2f17, that encodes an invalid variant value, and I'm not sure what that could break.

[UUID Decoder](https://www.uuidtools.com/decode)
[A Universally Unique IDentifier (UUID) URN Namespace](https://www.ietf.org/rfc/rfc4122.txt) `4.1.1.  Variant` gives interpretation of variable-bit-length field.
[Universally unique identifier](https://en.wikipedia.org/wiki/Universally_unique_identifier#Variants) Variants chose to number variants, but they call the `10` MSB variant 1 while others call it 2...

Looking at the current files, I see the new GUID has gotten around to other places, but I don't know process to correct it.
That's more a developer tool skill, so I'll leave it to developers to sort out. I just happen to know a bit about GUID format.

Reply:
Thanks @ts678.  I'd prefer that we do not manage the GUIDs manually.  @TopperDEL, can you fix the project references in your IDE?

Reply:
I've regenerated the GUID, so it is clearly unique.